tests/test-rest-posts-controller.php

```diff
@@ -420,19 +420,21 @@ public function test_get_items_pagination_headers() {
 		$this->assertContains( '<' . $prev_link . '>; rel="prev"', $headers['Link'] );
 		$this->assertFalse( stripos( $headers['Link'], 'rel="next"' ) );
 
-		// With filter params.
+		// With query params.
 		$request = new WP_REST_Request( 'GET', '/wp/v2/posts' );
-		$request->set_query_params( array( 'filter' => array( 'posts_per_page' => 5, 'paged' => 2 ) ) );
+		$request->set_query_params( array( 'per_page' => 5, 'page' => 2 ) );
```

Comment:
This was done because #2699 suppresses `filter[posts_per_page]` entirely, and we have first-party query parameters for pagination so it seems most future conscious to use them.